project/application/src/components/projectSearchBar/index.js

```diff
@@ -21,9 +21,12 @@ class ProjectSearchBar extends Component {
 
     search() {
         if (!!this.searchInput.value.length) {
-            let afterSearch = this.props.tableInfo.filter(obj =>
-                Object.values(obj).some(value => value === this.searchInput.value.toLowerCase())
-            );
+            let afterSearch = this.props.tableInfo.filter(obj => {
+                let objFiltered = Object.assign({}, obj, { id: undefined });
+                return Object.values(objFiltered).some(
+                    value => typeof value === 'string' && value.includes(this.searchInput.value.toLowerCase())
```

Comment:
What the purpose of using `typeof value === 'string'` ?

Btw `value` also needs to be in lowercase